projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -174,7 +174,7 @@ function page_render() {
 			// If we still have nothing, display an error
 			echo '<p>';
 			esc_html_e( 'Error fetching static.html. Try running: ', 'jetpack' );
-			echo '<code>yarn distclean && yarn build</code>';
+			echo '<code>pnpm run distclean && pnpm run build</code>';
```

Comment:
While we're here, should update this to the cli command.

```suggestion
			echo '<code>pnpm run distclean && pnpx jetpack build plugins/jetpack</code>';
```